src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/AbstractFormStore.js

```diff
@@ -1,15 +1,13 @@
 // @flow
-import {action, computed, isObservableArray, observable, set, toJS, untracked, when} from 'mobx';
+import {action, isObservableArray, observable, set, toJS} from 'mobx';
 import type {IObservableValue} from 'mobx';
-import jexl from 'jexl';
 import jsonpointer from 'json-pointer';
 import log from 'loglevel';
-import type {RawSchema, RawSchemaEntry, Schema, SchemaEntry} from '../types';
-import conditionDataProviderRegistry from '../registries/conditionDataProviderRegistry';
+import type {Schema, SchemaEntry} from '../types';
 
 const SECTION_TYPE = 'section';
 
-function addSchemaProperties(data: Object, key: string, schema: RawSchema) {
+function addSchemaProperties(data: Object, key: string, schema: Schema) {
```

Comment:
Strictly speaking yes, will add it.

Reply:
Actually... Wasn't the `RawSchema` used only internally? AFAIK the `schema` property of the `AbstractFormStore` was already returning a `Schema`. Also I don't want to spam the `UPGRADE.md` file too much, because people don't read it entirely already :see_no_evil:

Reply:
i dont know -  its okay for me to not document it as well. i really doubt that anybody relies on this change 🙂